evergreen.yml

```diff
@@ -37,7 +37,7 @@ tasks:
         set -o errexit
         . venv/bin/activate
 
-        poetry run pytest --flake8 --isort --junitxml=test_output_junit.xml
+        poetry run pytest --junitxml=test_output_junit.xml
```

Comment:
I think you can remove the `check_formatting` and `check_pydoc` tasks (below), since they're duplicated by the `unit_tests` task now.